frontend/awx/resources/inventories/InventoryGroups/InventoryGroupPage.tsx

```diff
@@ -0,0 +1,68 @@
+import { useTranslation } from 'react-i18next';
```

Comment:
Since Host can also have groups of inventories do you expect to be able to use this component inside of Hosts as well?

Reply:
Should I move the GroupPage to the resources/groups directory? I didn't realize this view will be used outside of invetories groups.

Reply:
So this view doesn't _have_ to be used outside of inventories.  The other possible location would be in host groups.   Maybe you could assign that work to yourself, and when you are done with inventories work you could explore reusing this component in Hosts > Groups.  This is probably the issue in question https://issues.redhat.com/browse/AAP-8160. I'll leave it up to you.

Reply:
I assigned it to myself and I'll add all the group details to the groups dir so it's be easy to use in both inventory and host.